swe_cp_WB.m

```diff
@@ -499,7 +498,7 @@ function swe_cp_WB(SwE)
       else
         descrip = sprintf('adjusted unrestricted residuals (%04d)', i);
       end
-      VResWB(i) = swe_create_vol(sprintf('ResWB_%04d.img', i), DIM, M, descrip);
+      VResWB(i) = swe_create_vol(sprintf('ResWB_%04d%s', i, file_ext), DIM, M, descrip);
```

Comment:
I didn't specifically consider this case... but maybe `swe_vox_resid_y{y#}` is OK here? (i.e. don't need to specifically note these are WB residuals... unless both WB and non-WB residuals co-exist)